controllers/activemqartemis_reconciler.go

```diff
@@ -2183,6 +2187,18 @@ func configurePodSecurityContext(podSpec *corev1.PodSpec, podSecurityContext *co
 	}
 }
 
+func configureContianerSecurityContext(container *corev1.Container, securityContext *corev1.SecurityContext) {
+	clog.V(1).Info("Configuring Container SecurityContext")
+
+	if nil != securityContext {
+		clog.V(5).Info("Incoming Container SecurityContext is NOT nil, assigning")
+		container.SecurityContext = securityContext
+	} else {
+		clog.V(5).Info("Incoming Container SecurityContext is nil, creating with default values")
+		container.SecurityContext = &corev1.SecurityContext{}
```

Comment:
If set, the fields of SecurityContext override the equivalent fields of PodSecurityContext, why is it set if customResource.Spec.DeploymentPlan.SecurityContext == nil